b2sdk/transfer/inbound/downloaded_file.py

```diff
@@ -165,6 +174,9 @@ def save(self, file, allow_seeking=True):
         :param allow_seeking: if False, download strategies that rely on seeking to write data
                               (parallel strategies) will be discarded.
         """
+        if file.mode == 'wb':
+            allow_seeking = False
```

Comment:
I don't think silently overwritting argument is a good idea, especially since whole thing can be easily overriden by us on CLI call for our purposes.

So it should other way around, i.e.

If `mode` wasn't set explicitly by the user, then: if `allow_seeking` is explicitly set by someone to `False`, we should use the default `wb`, and otherwise, use `wb+`.

Lets not forget our goal, which is to make download to `/dev/stdout` possible through CLI. And that is all.

In fact that, after looking at it more, it seems to me that this goal requires no changes to SDK, since we could very well do detection of "streamable" output file in CLI, and just call `.save_to(..., mode="wb", allow_seeking=False)`.

It seems all SDK changes can be forgone and this PR closed.

Reply:
I disagree. I think the sdk, when it sees a named pipe or a standard output (a device?), should switch just to a non-seekable output mode automatically... unless the user sets `allow_seeking` explicitly to `True` or `False`, in which case it's their fault if they configure it in a way that then doesn't work.